HealthVault/Classes/Extensions/MHVArrayExtensions.h

```diff
@@ -23,6 +23,12 @@
 
 + (BOOL)isNilOrEmpty:(NSArray *)array;
```

Comment:
I'm going to recommend we delete all methods in this file except for isNilOrEmpty. We need to clean up our extensions to include things that really make sense.

1. arrayByAddingObjectsFromArray:atStartingIndex: is used in the sample only, and should not be exposed publicly. 
2. arrayByRemovingObject: should just be done with a mutable array where needed.
3. hasElementsOfType: is not used anywhere.
4. addFromEnumerator: is not used anywhere.